Added New Year's Theme to themes.json and created required files

naeimehnajafi opened the pull request:
- Added the New Year's theme configuration to themes.json.
- Created `newyear.html`, `newyear.css`, and `newyear.js` files under the appropriate directory.
- Verified file paths and updated JSON configuration.

mads256h commented:
We already have a new years theme

JakobTopholt commented:
@mads256h I think she knows, the [ issue](https://github.com/f-klubben/stregsystemet/issues/546) related to this PR asks for a huge rocket on the moment of midnight.

mads256h commented:
@JakobTopholt I am just confused why a new theme is added

krestenlaust commented:
> @JakobTopholt I am just confused as to why a new theme is added

With the new theme system, themes can easily overlap, so you could just overlap the news rocket theme with the existing theme.

But with that said, it's a little silly to not build on top of the existing new year's rocket code

naeimehnajafi commented:
Hi,

I have integrated the rocket effect into the existing "fireworks" theme as per your guidance. The updates include:
- Adjustments in `fireworks.css` to incorporate the rocket visuals and animations.
- Updated `fireworks.html` to include the rocket’s `<div>` and logic.
- Verified and adjusted paths in `themes.md` and `themes.json` for consistency.
Let me know any issues.
Thank you for your help and patience!

krestenlaust commented:
Thank you for your interest in Stregsystemet. We might use your code for reference in the future, but will not be used for now. Thanks!
